File: test/fixtures/transforms/js-createandpoll-with-inline-create/expected.js
import OpenAI from "openai";

const openai = new OpenAI();

export async function askAssistant(assistantId, threadId, question) {
  const response = await openai.responses.create({
    model: "gpt-4-turbo",
    instructions: "Review pull requests and flag risky changes.",
    tools: [{ type: "file_search", vector_store_ids: ["vs_review_docs"] }],
    conversation: threadId,
    input: [{ role: "user", content: question }],
  });
  return response.output_text;
}

File: test/fixtures/transforms/python-async-dance/expected.py
import os

from openai import AsyncOpenAI

client = AsyncOpenAI()

ASSISTANT_ID = os.environ["OPENAI_ASSISTANT_ID"]


async def ask_assistant(prompt: str) -> str:
    thread = await client.conversations.create()
    response = await client.responses.create(
        conversation=thread.id,
        input=[{"role": "user", "content": prompt}],
    )
    return response.output_text
